### ISSUE TITLE
Drilldown to display specific documents from overview visualisation

### ISSUE THREAD

**Issue by kdutia** [climatepolicyradar/cpr-issues#113]
From the overview data visualisation, the user will be able to select a bubble and display a specific set of documents belonging to that bubble.

Tasks

- [x] @marcusodavies to provide a list of document ids
- [x] @kdutia update api to provide a route to return documents by id
- [x] @chrisaballard or @kdutia to write a script to look up the policy_id from the source ids provided by marcus, based on the new processed_policies file
- [ ] @eurolife to update frontend to call api and pass in document ids, to display matching documents

**Comment by chrisaballard** [comment]
@eurolife raised an issue with using the multiple document by id endpoint. Passing in some ids generates an internal server error. @kdutia to investigate.

**Comment by chrisaballard** [comment]
@eurolife I have fixed a bug with the `/policies/multiple' endpoint and raised pr 76 in policy_search repo. Missing ids are now ignored in the response, and are logged to the terminal for debug purposes.

I have noticed however that the ids provided for the bubble chart are incorrect, and need to be remapped to correct internal ids. All ids are either incorrect or are not in the dynamodb table. I have asked @marcusodavies to remap them, and once done code changes will be necessary to display the correct ids.
